src/command/backup/blockMap.h

```diff
@@ -0,0 +1,92 @@
+/***********************************************************************************************************************************
```

Comment:
This is a very compact representation of the block map when on disk. The limitation is the 20 byte checksum which can't be compressed. In memory, it takes around  48 bytes per entry, and I assume 25-30bytes when serialized to disk. Question: how many entries are we expecting the block maps to contain?  This is more doublechecking. From my first impressions, the map is reasonably sized with even a million entries.

Usually I would suggest ordering the fields by their alignment needs. It doesn't make any difference in this case, since SHA1 size is 20 bytes and all fields align without creating holes.

Reply:
Unless I've understated the likely size, this looks quite reasonable,